jira/client.py

```diff
@@ -970,15 +970,20 @@ def add_comment(self, issue, body, visibility=None):
             "type" is 'role' (or 'group' if the JIRA server has configured
             comment visibility for groups) and 'value' is the name of the role
             (or group) to which viewing of this comment will be restricted.
+        :param is_internal: defines whether a comment has to be marked as 'Internal' in Jira Service Desk
         """
         data = {
-            'body': body}
+            'body': body,
+            'properties':[{'key':'sd.public.comment','value':{'internal':is_internal}},]
```

Comment:
@ssbarnea, understood, did not know it will break other versions of JIRA as I tested only two of them (ServiceDesk) + Base JIRA (I forgot how it is called).

I will close the PR.